src/asmcnc/apps/start_up_sequence/start_up_sequence_manager.py

```diff
@@ -140,7 +140,7 @@ def prep_welcome_app(self):
 
 	def prep_release_notes_screen(self):
 		if not self.release_notes_screen:
-			self.release_notes_screen = screen_release_notes.ReleaseNotesScreen(name = 'release_notes', start_sequence = self, screen_manager = self.sm, localization = self.l, version = self.v)
+			self.release_notes_screen = screen_release_notes.ReleaseNotesScreen(name = 'release_notes', start_sequence = self, screen_manager = self.sm, localization = self.l, version = self.v, router_machine = self.m)
```

Comment:
Not needed if you get model_manager from its ModelManagerSingleton()

Reply:
I thought calling the singleton was slower than just passing an active instance. Am I mistaken?

Reply:
At latest with py3 and kivy 2.x we need to get rid of any kwargs that are not from kivy. Yes, it might be a bit slower for now but we can get around that with App.get_running_app().model_manager in a future refactoring.